src/models/node.js

```diff
@@ -100,9 +102,15 @@ MusitNode.pickNode = (pickNode$) => ({node, breadcrumb}) => {
   pickNode$.next({ value: node, path: breadcrumb });
 };
 
-MusitNode.findByBarcode = (ajaxGet = simpleGet) => ({ barcode, museumId, token }) => {
+MusitNode.findByBarcode = (ajaxGet = simpleGet) => ({ barcode, museumId, collectionId, token}) => {
```

Comment:
Arbeidsnotat for review:
Sjekker først om barcoden er en node, og hvis ikke, sjekker om det er et objekt